serverless.yml

```diff
@@ -21,22 +21,22 @@ provider:
             - "dynamodb:Scan"
           Resource: '*'
 
+custom:
+  alerts:
+    stages: # Optionally - select which stages to deploy alarms to
+      - dev     
```

Comment:
Why did you specify this? You don't want this alarm to be deployed in other stages?

Reply:
i only have this stage, so i did not really think about this, but i need to keep custom so i thought lets put something there and if i have more environments i just add them here.

Reply:
> Really nice! Congrats for the hard work!

i did, thank you very much!!